payments-ui-core/src/test/java/com/stripe/android/ui/core/elements/BacsDebitAccountNumberConfigTest.kt

```diff
@@ -0,0 +1,66 @@
+package com.stripe.android.ui.core.elements
+
+import androidx.compose.ui.text.input.KeyboardCapitalization
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.VisualTransformation
+import com.google.common.truth.Truth
+import com.stripe.android.uicore.elements.TextFieldStateConstants
+import org.junit.Test
+
+class BacsDebitAccountNumberConfigTest {
+    private val bacsDebitAccountNumberConfig = BacsDebitAccountNumberConfig()
+
+    @Test
+    fun `verify config uses proper visual transformation, keyboard capitalization, and keyboard type`() {
+        Truth.assertThat(
```

Comment:
Nit: Let’s statically import `assertThat` so that this is a little nicer to read.